func/upgrade.sh

```diff
@@ -632,7 +632,8 @@ upgrade_filemanager() {
 upgrade_roundcube(){
     if [ -n "$(echo "$WEBMAIL_SYSTEM" | grep -w 'roundcube')" ]; then
         if [ -d "/usr/share/roundcube" ]; then
-            echo "[ * ] Roundcube: Unable to update. Updates are managed by apt.";
+            echo "[ * ] Roundcube: Unable to update. Updates are managed over apt.";
```

Comment:
`[ ! ] Roundcube: Updates are currently managed using the apt package manager.` would sound better here.